pages.es/common/touch.md

```diff
@@ -0,0 +1,15 @@
+# touch
+
+> Cambia el tiempo de accesso y modificación de un archivo (atime, mtime).
+
+- Crea un archivo nuevo o cambia los tiempos de archivos existentes al tiempo actual:
+
+`touch {{archivo}}`
+
+- Establece los tiempos de un archivo a un dia y hora específicos:
+
+`touch -t {{YYYYMMDDHHMM.SS}} {{archivo}}`
+
+- Usa los tiempos de un archivo para establecer los tiempos en otro archivo:
+
+`touch -r {{archivo}} {{archivo2}}`
```

Comment:
We use long arguments when is possible:
```suggestion
`touch --reference={{archivo}} {{archivo2}}`
```

Reply:
In this case, it's **_not_** possible, so let's not do this. The original pages and all translations of it use the short argument, and one of the reasons for that is the ancient BSD `touch` bundled with macOS doesn't support it.
```console
$ touch file
$ touch --reference=file file2
touch: illegal option -- -
usage:
touch [-A [-][[hh]mm]SS] [-acfhm] [-r file] [-t [[CC]YY]MMDDhhmm[.SS]] file ...
$ sw_vers
ProductName:	macOS
ProductVersion:	11.2.2
BuildVersion:	20D80
$
```